.github/renovate.json

```diff
@@ -1,19 +1,25 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "docker": {
-    "enabled": false
-  },
-  "extends": ["github>redkubes/renovate-config"],
-  "helm-requirements": {
-    "enabled": false
-  },
-  "helm-values": {
-    "enabled": false
-  },
-  "helmfile": {
-    "enabled": false
-  },
+  "baseBranches": ["auto-chart-update"],
```

Comment:
Perhaps I need to add the `gitPrivateKey` to make the renovate working.

https://github.com/renovatebot/renovate/discussions/12692#discussioncomment-1651475